source/incqueryserver-jupyter/iqs_jupyter/config_defaults.py

```diff
@@ -23,11 +23,14 @@
 default_auto_display: bool = True
 default_use_oidc: bool = False
 default_use_password: bool = True
+default_use_auth_header: bool = False
```

Comment:
These 3 should be modified to use the `default_IQS_` prefix - after all, they relate to the connection to the IQS, as opposed to e.g. MMS. (probably we can keep the old names in deprecated form, to retain compatibility, and use their value if the newer environment variable name is not present)

Sorry for not catching this in earlier reviews.